serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/command/AxonCommandConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package io.axoniq.dev.samples.serializationavro.command;
+
+import org.axonframework.commandhandling.CommandBus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Profile("command")
+@Configuration
+public class AxonCommandConfiguration {
```

Comment:
If we drop the `ExceptionWrappingHandlerInterceptor`, I think we can drop this as well.

Reply:
Exactly.